src/cargo/ops/cargo_compile.rs

```diff
@@ -260,7 +260,8 @@ pub fn compile_ws<'a>(
         bail!("jobs must be at least 1")
     }
 
-    let mut build_config = BuildConfig::new(config, jobs, &target)?;
+    let rustc_info_cache = ws.target_dir().join("rustc_info.json").into_path_unlocked();
```

Comment:
Mind throwing a `.` on the front to hide it away?

Reply:
:heavy_check_mark: